salt/metalk8s/addons/prometheus-operator/deployed/kube-alerts-rules.sls

```diff
@@ -0,0 +1,48 @@
+#!jinja | metalk8s_kubernetes
+
+{%- from "metalk8s/repo/macro.sls" import build_image_name with context %}
+{%- set prometheus_defaults = salt.slsutil.renderer(
+        'salt://metalk8s/addons/prometheus-operator/config/prometheus.yaml',
+        saltenv=saltenv
+    )
+%}
+{%- set prometheus = salt.metalk8s_service_configuration.get_service_conf(
+        'metalk8s-monitoring', 'metalk8s-prometheus-config', prometheus_defaults
+    )
+%}
+{%- set rules = prometheus.get('spec', {}).get('rules', {}) %}
+
+{%- raw %}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    app: prometheus-operator
+    app.kubernetes.io/instance: prometheus-operator
+    app.kubernetes.io/managed-by: salt
+    app.kubernetes.io/name: prometheus-operator
+    app.kubernetes.io/part-of: metalk8s
+    app.kubernetes.io/version: 45.5.0
+    chart: kube-prometheus-stack-45.5.0
+    heritage: metalk8s
+    metalk8s.scality.com/monitor: ''
+    release: prometheus-operator
+  name: prometheus-operator-kubernetes-apps
+  namespace: metalk8s-monitoring
```

Comment:
fetched these from the chart. @TeddyAndrieux do any of them have to go ?

Reply:
Yes, make something closer to the `node-alerts-rules` one please (you shouldn't take the same name btw otherwise you will overwrite the other one)